charts/argo-workflows/values.yaml

```diff
@@ -776,6 +776,27 @@ server:
   # -- terminationGracePeriodSeconds for container lifecycle hook
   terminationGracePeriodSeconds: 30
 
+  ## livenessProbe for server
+  ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+  livenessProbe:
+    # -- Enable Kubernetes liveness probe for server
+    enabled: false
+    httpGet:
+      # -- Http port to use for the liveness probe
+      port: 2746
+      # -- Http path to use for the liveness probe
+      path: /
```

Comment:
I'm thinking out loud here, but this might be expensive. If server auth is enabled this would trigger quite a lot of k8s api calls to populate the tables etc.

There's no liveness/readiness probe upstream, right?

Ideally we need to find a quiet endpoint that'll just return a clean http 200